README.md

```diff
@@ -10,12 +10,12 @@ Some modern enterprise routers have the ability to stream telemetry in real-time
 
 The sample utilizes widely-used OSS tools for deployments. Ansible is used to configure virtual machines. Packer is used to automate creation and capture of VM images. Terraform is used for deployment of Azure resources.
 
-Required tools:
+Required tools w/ validated versions:
 
-* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
-* [Ansible](https://www.ansible.com/)
-* [Packer](https://https://www.packer.io/)
-* [Terraform](https://www.terraform.io/)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) `2.0.61`
+* [Ansible](https://www.ansible.com/) `2.8.0`
+* [Packer](https://https://www.packer.io/) `1.4.1`

```

Comment:
Oh also sorry - forgot to post this before but the Packer url is broken because we have 2 https!